hack/usb-iso.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+parse_args() {
+   WRITE_USB=${WRITE_USB:-true}
+
+   USAGE="Usage: $0 [options]
+Options:
+    -d USB_DEV     Install to the given usb device, alternatively
+                   specify env var USB_DEV
+    -i IGNITION    The URL or path to the given Ignition config,
```

Comment:
we need some feature, to indicate if we simply want to inject the URL, or we want to downloaded the generated ignition adn embed it into the initramfsExtra image